fury/ui.py

```diff
@@ -4269,6 +4269,240 @@ def directory_click_callback(self, i_ren, _obj, listboxitem):
+            reverse_scrolling=self.reverse_scrolling)
+
+        self.drop_down_menu.set_visibility(False)
+
+        self.panel = Panel2D(self.panel_size, self.panel_position, opacity=0.0)
+
+        self.panel.add_element(self.selection_box, (0.01, 0.3))
+        self.panel.add_element(self.drop_down_button, (0.7, 0.3))
+        self.panel.add_element(self.drop_down_menu, (0.01, 0.7))
+
+        self.add_callback(self.drop_down_menu.scroll_bar.actor,
+                          "MouseMoveEvent",
+                          self.scroll_callback)
+
+        up_event = "MouseWheelForwardEvent"
+        down_event = "MouseWheelBackwardEvent"
+        if self.reverse_scrolling:
+            up_event, down_event = down_event, up_event  # Swap events
+
+        self.add_callback(
+            self.drop_down_menu.panel.background.actor, up_event,
+            self.scroll_callback)
+
+        self.add_callback(
+            self.drop_down_menu.panel.background.actor, down_event,
+            self.scroll_callback)
+
+        # Handle mouse wheel events on the slots.
+        for slot in self.drop_down_menu.slots:
+            self.add_callback(slot.background.actor, up_event,
+                              self.scroll_callback)
+            self.add_callback(slot.background.actor, down_event,
+                              self.scroll_callback)
+            self.add_callback(slot.textblock.actor, up_event,
+                              self.scroll_callback)
+            self.add_callback(slot.textblock.actor, down_event,
+                              self.scroll_callback)
+
+            slot.add_callback(
+                slot.textblock.actor, "LeftButtonPressEvent",
+                self.select_option_callback)
+
+            slot.add_callback(
+                slot.background.actor, "LeftButtonPressEvent",
+                self.select_option_callback)
+
+            self.drop_down_button.on_left_mouse_button_clicked = \
+                self.menu_toggle_callback
+
+    def _get_actors(self):
+        """ Get the actors composing this UI component.
+        """
+        return self.panel.actors
+
+    def resize(self):
+        pass
+
+    def _set_position(self, coords):
+        """ Position the lower-left corner of this UI component.
+
+        Parameters
+        ----------
+        coords: (float, float)
+            Absolute pixel coordinates (x, y).
+        """
+        self.panel.position = coords
+
+    def _add_to_scene(self, scene):
+        """ Add all subcomponents or VTK props that compose this UI component.
+
+        Parameters
+        ----------
+        scene : scene
+        """
+        self.panel.add_to_scene(scene)
+
+    def _get_size(self):
+        return self.panel.size
+
+    @property
+    def getCurrentText(self):
+        return self._selection
+
+    @property
+    def getCurrentIndex(self):
+        return self._selection_ID
+
+    @property
+    def appendItem(self, item):
```

Comment:
1. it should not be a property. 
2. `append_item` instead
3. `*items` as parameters

Reply:
Yes!! My bad